MarkdownChart/rsbuild.config.ts

```diff
@@ -20,19 +20,19 @@ export default defineConfig({
     port: 3005,
   },
   dev: {
-    assetPrefix: '/plugins-dev/Markdown/',
+    assetPrefix: '/plugins-dev/MarkdownChart/',
   },
   output: {
-    assetPrefix: '/plugins/Markdown/',
+    assetPrefix: '/plugins/MarkdownChart/',
   },
   plugins: [pluginReact()],
   tools: {
     htmlPlugin: false,
     rspack: (config, { appendPlugins }) => {
-      config.output!.uniqueName = 'Markdown';
+      config.output!.uniqueName = 'MarkdownChart';
       appendPlugins([
         new ModuleFederationPlugin({
-          name: 'Markdown',
+          name: 'MarkdownChart',
```

Comment:
I wanted to match the plugin kind name, so it works correctly with a dashboard definition when being loaded. https://github.com/perses/perses/blob/main/ui/panels-plugin/src/plugins/markdown/markdown-panel-model.ts#L20 We could rename this but we would need to adjust elsewhere so the plugin still works. For example here: https://github.com/perses/perses/blob/feature/plugin-runtime/ui/dashboards/src/components/DynamicPanel/DynamicPanelContent.tsx#L30

Reply:
yup, that sounds good to me as long you are ok with it as well.

Reply:
I'm preparing a PR in Perses, reviewing a bit how to install a plugin with the backend involved. Will fix that in the meantime

Reply:
I'm ok to be more consistent with the plugin names, but this will mean the current dashboards using `Markdow` need to be migrated to use `MarkdownChart`, right?

Reply:
yes migration will be needed

Reply:
that's definitively a breaking change